### PR TITLE
Preload models

### PR DESCRIPTION
**Pull request by ventus550**
Feed models with dummy tensors to speed up the first inference.

### CODE REVIEW

File: src/services/magic/TensorflowWorker.ts [hunk 1]
```diff
@@ -34,6 +34,8 @@ tf.loadLayersModel(modelUrl).then((model) => {
   }
 
   logger.debug('Model initialized!', model)
+  const shape = model.inputs[0].shape.slice(1)
+  if (!shape) model.call(tf.zeros([1, ...shape]), {})
```
Comment (MusicFreak456) [inline src/services/magic/TensorflowWorker.ts:38]: We should have a mechanism for awaiting the end of the initialization in context of implementing proper page loading screen in the future. Perhaps reverting to AsyncInitialized pattern is in place.

**Review by MusicFreak456 — CHANGES_REQUESTED** [review]
Async init needs to be addressed

**Review by MusicFreak456 — APPROVED** [review]

**Comment by MusicFreak456** [comment]
Awaiting review from @ventus550

File: src/components/whiteboard/magic/HandwritingRecognition.vue [hunk 2]
```diff
@@ -140,6 +140,7 @@ watch(correctionRequestState, (nextState, previousState) => {
 
 onMounted(async () => {
   await handwritingRecognizer.init()
+  logger.debug('Hand recognition initialized!')
```
Comment (ventus550) [inline src/components/whiteboard/magic/HandwritingRecognition.vue:143]: "Handwriting recognition" please. Hand recognition would be much less of an interesting problem :P

**Comment by ventus550** [comment]
The changes are necessary, however unfortunate they may appear. :ship:

File: src/components/whiteboard/magic/HandwritingRecognition.vue (hunk 2 above)
Comment (MusicFreak456) [reply src/components/whiteboard/magic/HandwritingRecognition.vue]: Done

**Pull request merged**